### PR TITLE
User search page#5

### LINKED ISSUE #5: User Search
As a logged in user
  * My current path is /search
  * I see a form with fields
	Location
    * Zip code or Use current location
    * Enter your range (Dropdown in 5 mi increments)
  Categories
    * Option form (TBD)

### PR DESCRIPTION
**Pull request by mgoodhart5**
closes #5 
As a logged in user
my current path is /search
I see a form with fields
Location
Zip code (or Use current location to be implemented further down the line)
Enter your range (Dropdown in 5 mi increments)
Categories
Option form (TBD)
paired branch with @Bradniedt

### CODE REVIEW

File: app/views/search/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<%= form_tag :search_index, method: :get do %>
+  <p>Your EV Range (in miles)<%= select_tag :ev_range, options_for_select(ranges) %></p>
```
Comment (stoic-plus) [inline app/views/search/new.html.erb]: I have routes for this - I would recommend changing this to a POST that routes to a `search#create`

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def ranges
+    [5, 10, 15, 20, 25, 30, 35, 40, 45, 50]
+  end
+
+  def activities
+    ["Food", "Culture", "Entertainment", "Outdoors"]
+  end
```
Comment (stoic-plus) [inline app/helpers/application_helper.rb]: Way to compartmentalize! Very clean.

File: app/views/search/new.html.erb (hunk 1 above)
Comment (mgoodhart5) [reply app/views/search/new.html.erb]: will make changes and push again.

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (mgoodhart5) [reply app/helpers/application_helper.rb]: Thanks!!

**Pull request merged**